inat.label.py

```
@@ -803,7 +803,7 @@ def _parse_retry_after(resp: requests.Response) -> float | None:
 _TAXON_BATCH_MAX = int(os.environ.get("INAT_TAXON_BATCH_MAX", "50"))
 _TAXON_BATCH_WINDOW = float(os.environ.get("INAT_TAXON_BATCH_WINDOW", "0.1"))
 # How often (seconds) to check batcher liveness while waiting for a taxon event.
-# There is no upper time limit on the wait itself — In normal operation, the batcher’s 
+# There is no upper time limit on the wait itself — In normal operation, the batcher’s
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)